.editorconfig

```diff
@@ -9,3 +9,6 @@ indent_style = space
 endofline = lf
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[*.json]
+indent_size = 2
```

Comment:
Here we go again :)

How about setting this explicitly to 4, so that your editor won't try to overwrite it again?